mlc-llm/tutorial_chat_module_getting_started.ipynb

```diff
@@ -1,549 +1 @@
-{
```

Comment:
<p>Does it still work if we remove the <code>device_name</code> as well? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlc-ai/notebooks/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
(I was trying to reference the `ChatModule()` call in the getting started notebook)

Reply:
Yes, removing the device name does indeed still pickup the correct cuda library, let me update that in the next iteration.